frontend/app/src/components/account-management/LoginForm.vue

```diff
@@ -434,19 +438,68 @@ const login = async (syncApproval: SyncApproval = 'unknown') => {
 
               <v-row justify="end" class="mt-2">
                 <v-col cols="auto" class="shrink">
-                  <v-btn color="error" depressed @click="login('no')">
+                  <v-btn
+                    color="error"
+                    depressed
+                    @click="login({ syncApproval: 'no' })"
+                  >
                     {{ t('common.actions.no') }}
                   </v-btn>
                 </v-col>
                 <v-col cols="auto" class="shrink">
-                  <v-btn color="success" depressed @click="login('yes')">
+                  <v-btn
+                    color="success"
+                    depressed
+                    @click="login({ syncApproval: 'yes' })"
+                  >
                     {{ t('common.actions.yes') }}
                   </v-btn>
                 </v-col>
               </v-row>
             </v-alert>
           </transition>
 
+          <transition>
+            <v-alert
+              v-if="!!halfUpgradeConflict.message"
+              class="animate half__upgrade-error mt-8"
```

Comment:
should this be `half-upgrade-error` also maybe instead of half it would make sense to use `partial`